### PR TITLE
Create footer section

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72044405
image

### CODE REVIEW

File: practice-one/src/styles/components/footer.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  margin: 37px 0 27px 0;
+  font-size: var(--font-size-normal, 18px);
+}
+
+.subscribe-input > input[type="text"] {
+  width: 205px;
+  height: 50px;
+  border-radius: 38px;
+  padding-left: 20px;
+  background: var(--color-neutral, #C4C4C4);
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/footer.css]: recheck this value, do we have variable for this

File: practice-one/src/components/footer.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    <div class="company-information-container">
+      <h3 class="company-information-heading">Company</h3>
+      <ul class="company-information-list">
+        <li class="company-information-item"><a href="">Partnership</a></li>
+        <li class="company-information-item"><a href="">Terms of Use</a></li>
+        <li class="company-information-item"><a href="">Privacy</a></li>
+        <li class="company-information-item"><a href="">Sitemap</a></li>
+      </ul>
+    </div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/footer.html]: duplicate styling, please check

File: practice-one/src/components/footer.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    <form class="subscribe-form" action="URL" method="post">
```
Comment (hungdo-agilityio) [inline practice-one/src/components/footer.html]: `action="URL" method="post"` what this for

File: practice-one/src/components/footer.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<footer class="footer-container">
+  <div class="website-summary-container">
+    <div class="brief-description">
+      <h3 class="brief-heading">Foodhut</h3>
+      <p class="brief-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor </p>
+      <div class="brief-icon-container">
+        <div class="facebook-icon"><img src="./assets/images/facebook.svg" alt="Facebook icon"></div>
+        <div class="instagram-icon"><img src="./assets/images/instagram.svg" alt="Instagram icon"></div>
+        <div class="twitter-icon"><img src="./assets/images/twitter.svg" alt="Twitter icon"></div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/footer.html]: duplicate styling

File: practice-one/src/styles/components/footer.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  width: 325px;
+  padding-left: 56px;
+  margin-right: 109px;
+}
+
+.brief-heading {
+  color: var(--color-primary,#F54748);
+  font-weight: var(--font-weight-sub-heading, 600);
+  line-height: var(--line-height-normal, 1.3);
+  font-size: var(--font-size-extra-large, 32px);
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/footer.css]: recheck for all

File: practice-one/src/components/footer.html (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/components/footer.html]: i fixed it , thanks

File: practice-one/src/components/footer.html (hunk 3 above)
Comment (CBinh710) [reply practice-one/src/components/footer.html]: it is unecessary in this case, so i deleted it, Thanks for your contribution

File: practice-one/src/components/footer.html (hunk 4 above)
Comment (CBinh710) [reply practice-one/src/components/footer.html]: i changed it

File: practice-one/src/styles/components/footer.css (hunk 5 above)
Comment (CBinh710) [reply practice-one/src/styles/components/footer.css]: I think value after (,) is unecessary too but when i check it in css validator it displayed warning. Can you make clear this issue 
image

File: practice-one/src/styles/components/footer.css (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/styles/components/footer.css]: I added a variable for this

File: practice-one/src/components/footer.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+    </div>
+
+    <div class="company-information-container">
+      <h3 class="company-information-heading">About Us</h3>
+      <ul class="company-information-list">
+        <li class="company-information-item"><a href="">About Us</a></li>
+        <li class="company-information-item"><a href="">Service Us</a></li>
+        <li class="company-information-item"><a href="">Contact</a></li>
+        <li class="company-information-item"><a href="">Company</a></li>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/footer.html:19]: Missing href, any idea for this? You can provide a mock value for this and comment above for clear

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**